Automatic Validation ended with: 
> 2026-04-25T12:06:35.9362037Z ##[error] Manifest Error: Field usage requires verified publishers. [Icons]
2026-04-25T12:06:35.9363811Z ##[error] Manifest Error: Field usage requires verified publishers. [Icons]

(Automated response - build 1260.)

--- a/manifests/u/ubidzz/Synix/1.0.11-beta/ubidzz.Synix.locale.en-US.yaml
+++ b/manifests/u/ubidzz/Synix/1.0.11-beta/ubidzz.Synix.locale.en-US.yaml
@@ -11,7 +11,7 @@ Author: Jason Turner
 PackageName: Synix Control Panel
 PackageUrl: https://github.com/ubidzz/Synix-Control-Panel
 License: All Rights Reserved
-LicenseUrl: https://github.com/ubidzz/Synix-Control-Panel/blob/main/README.md
+LicenseUrl: https://github.com/ubidzz/Synix-Control-Panel/blob/master/README.md
 Copyright: Copyright 2026 Jason Turner. All Rights Reserved.
 ShortDescription: Synix is a control panel that allows users to host their own game servers on Windows.
 Description: Synix is an easy point-and-click game server control panel to allow any user to host their own game server on their computer.

--- a/manifests/u/ubidzz/Synix/1.0.11-beta/ubidzz.Synix.locale.en-US.yaml
+++ b/manifests/u/ubidzz/Synix/1.0.11-beta/ubidzz.Synix.locale.en-US.yaml
@@ -25,8 +25,5 @@ Tags:
 - synix
 ReleaseNotes: https://github.com/ubidzz/Synix-Control-Panel/releases/tag/v1.0.11-beta
 ReleaseNotesUrl: https://github.com/ubidzz/Synix-Control-Panel/releases/tag/v1.0.11-beta
-Icons:
-- IconUrl: https://raw.githubusercontent.com/ubidzz/Synix-Control-Panel/master/Images/SyniX%20logo.ico
-  IconFileType: ico
 ManifestType: defaultLocale
 ManifestVersion: 1.6.0

